pallets/liquid-staking/src/lib.rs

```diff
@@ -326,6 +329,11 @@ pub mod pallet {
                 T::BaseXcmWeight::get(),
             )?;
 
+            ensure!(
+                matches!(xcm_result, Outcome::Complete(_)),
+                Error::<T>::XcmTransferFailed
```

Comment:
If xcm transfer failed, does the fund returns back to the original account?

Reply:
@kaichaosun it depends, if it fails on parachain, then yes, but if it fails on relaychain like:

```
FailedToTransactAsset 
```

then we lose them